src/adapter/src/coord/message_handler.rs

```diff
@@ -321,6 +324,11 @@ impl Coordinator {
         }
     }
 
+    #[mz_ore::instrument(level = "debug")]
+    async fn storage_usage_prune(&mut self, expired: Vec<BuiltinTableUpdate>) {
+        self.builtin_table_update().execute(expired).await.await
```

Comment:
The dreaded double `.await.await` ... 🙈

Reply:
I guess we should probably do the second await in a separate task so we don't block the main thread.